docs/user/tutorial/60-admin.rst

```diff
@@ -45,10 +47,29 @@ Add an image to the Filer
 .. todo:: write up steps properly
 
 * hit New folder in Filer, create folder called "People"
+
+.. image:: images/new_folder.png
+   :alt: create folder in filer
+   :width: 80%
+
 * open folder
+
+.. image:: images/open_folder.png
+   :alt: open the new folder
+   :width: 80%
+
 * hit upload button
+
+.. image:: images/upload_filer.png
```

Comment:
Can we also have an image of the upload button itself, so it can be added inline in the text please?